src/frontend/apps/calendars/src/features/calendar/components/scheduler/scheduler-theme.scss

```
@@ -261,10 +261,48 @@
 }
 
 // -----------------------------------------------------------------------------
-// 9. Mobile adjustments - Hide native day headers (WeekDayBar replaces them)
+// 9. Mobile adjustments — single-day view only.
+//
```

Stylelint (17.12.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)